1-js/02-first-steps/15-function-basics/article.md

```diff
@@ -247,7 +247,7 @@ function showMessage(text) {
 showMessage(); // empty message
 ```
 
-...یا می‌توانستیم از عملگر `||` استفاده کنیم:
+...یا می‌توانیم از عملگر `||` استفاده کنیم:
```

Comment:
These are really minor (if even) problems, I suggest try translating untranslated articles (#1, [readme](https://github.com/javascript-tutorial/fa.javascript.info#readme)), instead of looking for these minor problems.